### PR TITLE
improving_docs

### PR DESCRIPTION
**Pull request by jposada202020**
@dhalbert This one is a little different, I would like some feedback on how to deal with the different speed in the buses. I have not change yet the busio reference waiting for your feedback, however all the rest is done. If you could guide me, there is also the BNO08x_RVC that is setup in the same way, 

Thanks,

### CODE REVIEW

File: adafruit_bno08x/i2c.py [hunk 1]
```diff
@@ -17,6 +16,47 @@ class BNO08X_I2C(BNO08X):
     """Library for the BNO08x IMUs from Hillcrest Laboratories
 
     :param ~busio.I2C i2c_bus: The I2C bus the BNO08x is connected to.
+    :param ~digitalio.DigitalInOut reset: The pin object to reset
```
Comment (dhalbert) [inline adafruit_bno08x/i2c.py:19]:
```suggestion
    :param ~digitalio.DigitalInOut reset: Optional for I2C use. Connected to the RST pin; used to hard-reset the device.
```

File: examples/bno08x_simpletest_spi.py [hunk 2]
```diff
@@ -3,12 +3,11 @@
 # SPDX-License-Identifier: Unlicense
 from time import sleep
 import board
-import busio
 from digitalio import DigitalInOut, Direction
 from adafruit_bno08x.spi import BNO08X_SPI
 
 # need to limit clock to 3Mhz
```
Comment (dhalbert) [inline examples/bno08x_simpletest_spi.py:9]:
```suggestion
# need to limit clock to 3MHz
```

File: adafruit_bno08x/spi.py [hunk 3]
```diff
@@ -15,20 +14,54 @@
...
 
-    Args:
-        spi_bus ([busio.SPI]): The SPI bus to use to communicate with the BNO08x
-        cs_pin ([digitalio.DigitalInOut]): The pin object to use for the SPI Chip Select
-        debug (bool, optional): Enables print statements used for debugging. Defaults to False.
-    """
+    :param ~busio.SPI spi_bus: The SPI bus to use to communicate with the BNO08x
+    :param ~digitalio.DigitalInOut cspin: The pin object to use for the SPI Chip Select
+    :param ~digitalio.DigitalInOut intpin: The pin object to interrupt
+    :param ~digitalio.DigitalInOut resetpin: The pin object to reset
```
Comment (dhalbert) [inline adafruit_bno08x/spi.py:24]:
```suggestion
    :param ~digitalio.DigitalInOut resetpin: Required for SPI mode. Connected to the RST pin on the device, and used to hard-reset the device.
```

File: adafruit_bno08x/__init__.py [hunk 4]
```diff
@@ -483,7 +483,11 @@ def is_error(cls, header):
 class BNO08X:  # pylint: disable=too-many-instance-attributes, too-many-public-methods
     """Library for the BNO08x IMUs from Hillcrest Laboratories
 
-    :param ~busio.I2C i2c_bus: The I2C bus the BNO08x is connected to.
+    This sensor could use three different communication buses:
```
Comment (dhalbert) [inline adafruit_bno08x/__init__.py:486]:
```suggestion
    This sensor is able to communicate over I2C, SPI, or UART:
```

File: adafruit_bno08x/i2c.py [hunk 5]
```diff
@@ -17,6 +16,47 @@ class BNO08X_I2C(BNO08X):
...
+
+            import board
+            import busio
+            from adafruit_bno08x.i2c import BNO08X_I2C
+
+        Once this is done you can define your `busio.I2C` object and define your sensor object
+
+        .. code-block:: python
+
+            i2c = busio.I2C(board.SCL, board.SDA, frequency=400000)
```
Comment (dhalbert) [inline adafruit_bno08x/i2c.py:39]:
```suggestion
           # The sensor can communicate over I2C at 400kHz if you need the higher speed.
            i2c = busio.I2C(board.SCL, board.SDA, frequency=400000)
```

**Review by dhalbert — CHANGES_REQUESTED** [review]
I did some more copy-editing. My impression is that 400kHz is not required, but it can run at the higher speed, so I added a comment to that effect.

**Comment by dherrada** [comment]
Nice job getting the docs to build!

**Comment by jposada202020** [comment]
Danh let me know if you have any other comments, gladly to work on that, if `None` I think that we could make the changes. Thanks

**Review by dhalbert — APPROVED** [review]
Sorry - this fell off my list. Looks good!

**Comment by dherrada** [comment]
@jposada202020 Do you think you could reopen this branch and make another PR? I messed something up with moving to main and this PR wasn't included in the main branch.

**Comment by jposada202020** [comment]
@dherrada No problem I could that tonight. Just to clarify, I need to resubmit PR? to main, is that correct? just to be sure and apply the same for all the others :) thanks

**Comment by dherrada** [comment]
@jposada202020 Yep. Luckily the mistake that I made seems to not have affected many libraries, but 2 or 3 PRs you made were affected and I've pinged you in all of them. I'm not really sure how late you can reopen a branch or if you have it saved locally, so let me know if you've got questions.

**Comment by jposada202020** [comment]
@dherrada Ok, Understood, depends on how often I work in the library, I have or have not the branch. But I will figure it out. And make the changes accordingly.

**Comment by dherrada** [comment]
@jposada202020 Sounds good.

**Pull request merged**